steward/src/commands/cork_proposal/aave_v2_stablecoin/sweep.rs

```diff
@@ -0,0 +1,68 @@
+use crate::{application::APP, cellars, commands::cork_proposal::print_proposal, prelude::*};
+use abscissa_core::{clap::Parser, Command, Runnable};
+use ethers::types::Address;
+use steward_proto::steward::{
+    aave_v2_stablecoin_governance::{Function, Sweep},
+    governance_call::Call,
+    AaveV2StablecoinGovernance, GovernanceCall,
+};
+
+/// Sweep subcommand
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION\n\nSchedule sweep Cosmos address of target cellar when chain reaches specified height"
```

Comment:
If we're linking to the contract in the `CellarV1` commands, should we do it for Aave as well?